scripts/postprocess.py

```diff
@@ -190,9 +190,15 @@ def _calc_metrics(cols, df):
 #     "nodes":
 # }
 def _parse_line_based(fout, bench_name, nodes):
-    if "dlaf" in bench_name:
+    if bench_name.startswith("chol_dlaf"):
         pstr_arr = []
-        pstr_res = "[{run_index:d}] {time:g}s {perf:g}GFlop/s ({matrix_rows:d}, {matrix_cols:d}) ({block_rows:d}, {block_cols:d}) ({grid_rows:d}, {grid_cols:d}) {:d}"
+        pstr_res = "[{run_index:d}] {time:g}s {perf:g}GFlop/s {} {} ({matrix_rows:d}, {matrix_cols:d}) ({block_rows:d}, {block_cols:d}) ({grid_rows:d}, {grid_cols:d}) {:d} {}"
+    elif bench_name.startswith("trsm_dlaf"):
+        pstr_arr = []
+        pstr_res = "[{run_index:d}] {time:g}s {perf:g}GFlop/s {} {} {} {} {} ({matrix_rows:d}, {matrix_cols:d}) ({block_rows:d}, {block_cols:d}) ({grid_rows:d}, {grid_cols:d}) {:d} {}"
+    elif "dlaf" in bench_name:
+        pstr_arr = []
+        pstr_res = "[{run_index:d}] {time:g}s {perf:g}GFlop/s {} ({matrix_rows:d}, {matrix_cols:d}) ({block_rows:d}, {block_cols:d}) ({grid_rows:d}, {grid_cols:d}) {:d} {}"
```

Comment:
I'm not sure this is the best way as a comparison of old/new data is not possible without modifying the outputs.
I suggest to use a single compact string to identify the type and the blas options (e.g. dL, zU, ... for Cholesky), and should be used to separate the results (e.g. it should be possible to filter only double precision results).
Moreover, this string should be optional such that old outputs can be loaded.

Reply:
I've updated the miniapps to print a single compact string for the matrix type now. To help with this I've added a `FormatShort` helper struct that simple wraps an existing value, and can be used to overload `operator<<`. This way the default is the long formatting and a more compact representation can be opted into explicitly. The element type is also case insensitive now.

`parse` doesn't support optional fields directly but it does support parsing custom types. Specifying `(|\s+\S+)` as the regex for a custom type matches nothing or whitespace followed by non-whitespace. This seems to work fine at least for the old and new format and I hope there are no funny edge cases...

Reply:
@rasolca are you happy with the update?

Reply:
Yes.